### PR TITLE
20 cleaning services

### PR DESCRIPTION
**Pull request by BovkunLeonid**

### CODE REVIEW

File: Cleaning_Services/index.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="build/styles.css">
+</head>
+<body>
+    <header>
+        <div class="containera">
```
Comment (RomanIlkiv) [inline Cleaning_Services/index.html:21]: typo in name of class
`container` is the correct class name of Bootstrap

File: Cleaning_Services/index.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                        <h1 class="logo-name">Round Rock</h1>
+                        <p class="logo-p">Cleaning Services</p>
+                    </a>
+                </div>
+                <div class="lorem">
+                    <p>Lore m ipsum Lorem</p>
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled
+                    </p>
+                </div>
+                <div class="checkbox-groop">
```
Comment (RomanIlkiv) [inline Cleaning_Services/index.html:42]: typo again
`group`

File: Cleaning_Services/index.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                    <span class="border-button"></span>
+                </button>
+            </div>
+        </div> 
+    </header>
+    <section class="about-us">
+        <h2 class="title">About Us</h2>
+        <div class="flex-wrap">
+            <div class="wraper-about">
+                <div class="namber">1</div>
```
Comment (RomanIlkiv) [inline Cleaning_Services/index.html:69]: `number`

File: Cleaning_Services/index.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+            <div class="wraper-about">
+                <div class="namber">5</div>
+                <p class="about-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley</p>
+            </div>
+            <div class="wraper-about">
+                <div class="namber">6</div>
+                <p class="about-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley</p>
+            </div>
+        </div>
+        <button type="button" class="button-style button-centr">I need help
```
Comment (RomanIlkiv) [inline Cleaning_Services/index.html:93]: `center`

File: Cleaning_Services/styles/_about.scss [hunk 5]
```diff
@@ -0,0 +1,62 @@
+.title {
+    margin-top: 50px;
+    @include midle-text;
```
Comment (RomanIlkiv) [inline Cleaning_Services/styles/_about.scss:3]: good job!

File: Cleaning_Services/index.html [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="build/styles.css">
+</head>
+<body>
+    <header>
+        <div class="containera">
+            <div class="contact-us">
+                <h2 class="cont">Contact Us</h2>
+                <input type="text" class="form-control input-cont" placeholder="Name">
+                <input type="text" class="form-control input-cont" placeholder="telephone">
```
Comment (RomanIlkiv) [inline Cleaning_Services/index.html:25]: it's better to write placeholders from capital letter

File: Cleaning_Services/index.html [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+                <input type="email" class="form-control input-cont" placeholder="email">
+                <button class="butt" type="button"></button>
```
Comment (RomanIlkiv) [inline Cleaning_Services/index.html:27]: add some text inside of button to not make it empty:)

File: Cleaning_Services/styles/_general.scss [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+    font-weight: 500;
+    line-height: 24px;
+
+    &:hover {
+        background: #2E8CAD;
+        border: 1px solid #2E8CAD;  
+    } 
+    
+    &:active {
+        transform: translate(5px, 5px);
```
Comment (RomanIlkiv) [inline Cleaning_Services/styles/_general.scss:32]: good job!

**Review by RomanIlkiv — COMMENTED** [review]
very very good job!
well done!

File: Cleaning_Services/styles/_general.scss (hunk 8 above)
Comment (BovkunLeonid) [reply Cleaning_Services/styles/_general.scss:32]: Thank you very much

File: Cleaning_Services/index.html (hunk 7 above)
Comment (BovkunLeonid) [reply Cleaning_Services/index.html:27]: Thank you very much

File: Cleaning_Services/index.html (hunk 6 above)
Comment (BovkunLeonid) [reply Cleaning_Services/index.html:25]: Thank you very much

File: Cleaning_Services/styles/_about.scss (hunk 5 above)
Comment (BovkunLeonid) [reply Cleaning_Services/styles/_about.scss:3]: Thank you very much

File: Cleaning_Services/index.html (hunk 4 above)
Comment (BovkunLeonid) [reply Cleaning_Services/index.html:93]: Thank you very much

File: Cleaning_Services/index.html (hunk 3 above)
Comment (BovkunLeonid) [reply Cleaning_Services/index.html:69]: Thank you very much

File: Cleaning_Services/index.html (hunk 2 above)
Comment (BovkunLeonid) [reply Cleaning_Services/index.html:42]: Thank you very much

**Pull request merged**